pkg/discovery/dns/miekgdns/resolver.go

```diff
@@ -50,6 +50,15 @@ func (r *Resolver) LookupSRV(ctx context.Context, service, proto, name string) (
 }
 
 func (r *Resolver) LookupIPAddr(ctx context.Context, host string) ([]net.IPAddr, error) {
+	return r.lookupIPAddr(ctx, host, 1, 8)
+}
+
+func (r *Resolver) lookupIPAddr(ctx context.Context, host string, currIteration, maxIterations int) ([]net.IPAddr, error) {
```

Comment:
Can we remove the context from the signature entirely since it's not used?

Reply:
Definitely, done!